src/main/java/org/opentripplanner/routing/api/request/preference/MappingFeature.java

```diff
@@ -0,0 +1,27 @@
+package org.opentripplanner.routing.api.request.preference;
+
+import org.opentripplanner.framework.doc.DocumentedEnum;
+import org.opentripplanner.routing.algorithm.mapping.RaptorPathToItineraryMapper;
+
+/**
+ * Opt-in features for {@link RaptorPathToItineraryMapper}.
```

Comment:
I think we discussed previously that we should avoid imports from different package structures just for linking. So instead use the full class path for linking.